longestPalindrome.js

```diff
@@ -0,0 +1,27 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/longest-palindrome/
+
+const longestPalindrome = function (s) {
+  const hashMap = {};
+  for (let i = 0; i < s.length; i += 1) {
```

Comment:
For cases like you have here I think it's simpler to iterate over **array** (or **string**) as: `for ( variable of iterableObjectName)`,

see https://www.geeksforgeeks.org/how-to-iterate-over-characters-of-a-string-in-javascript/ for details.